### PR TITLE
FB-1223 Add matcher for segmentation violation

### PR DESCRIPTION
**Pull request by ppp0**

### CODE REVIEW

File: modules/php5/manifests/extension/newrelic.pp [hunk 1]
```diff
@@ -28,6 +28,14 @@
     read_from_head => true,
   }
 
+  @fluentd::config::source_logfile { 'newrelic-php-agent-segfaults':
+    path        => '/var/log/newrelic/php_agent.log',
```
Comment (zazabe) [inline modules/php5/manifests/extension/newrelic.pp]: formatting

**Comment by zazabe** [comment]
```
Running php5:extension:newrelic for Debian-8
vagrant status Debian-8
vagrant snapshot list Debian-8
vagrant snapshot restore Debian-8 default --no-provision
vagrant ssh-config Debian-8 > /tmp/20170822-11429-1a5en0y
Puppet applying: `/vagrant/modules/php5/spec/extension/newrelic/2-segfault.pp`
Warning: /etc/hiera.yaml: Use of 'hiera.yaml' version 3 is deprecated. It should be converted to version 5
   (in /etc/hiera.yaml)
Notice: Compiled catalog for debian-8-amd64-plain.vagrantup.com in environment production in 0.28 seconds
Notice: /Stage[main]/Main/Node[default]/Helper::Script[run simple php script and segfault it]/Exec[exec run simple php script and segfault it]/returns: ./749d71b922190d33e0314e614b1ecbd3: line 3: php: command not found
Notice: /Stage[main]/Main/Node[default]/Helper::Script[run simple php script and segfault it]/Exec[exec run simple php script and segfault it]/returns: kill: usage: kill [-s sigspec | -n signum | -sigspec] pid | jobspec ... or kill -l [sigspec]
Error: '
mkdir -p /tmp/749d71b922190d33e0314e614b1ecbd3 && \
cd /tmp/749d71b922190d33e0314e614b1ecbd3 && \
echo \#\!/bin/bash'
''
'\(php\ -r\ \'sleep\(10\)\;\'\ \&\)\ \&\&\ sleep\ 1\ \&\&\ kill\ -11\ \$\(pgrep\ php\)'
' > 749d71b922190d33e0314e614b1ecbd3 && \
chmod +x 749d71b922190d33e0314e614b1ecbd3 && \
./749d71b922190d33e0314e614b1ecbd3' returned 2 instead of one of [0]
Error: /Stage[main]/Main/Node[default]/Helper::Script[run simple php script and segfault it]/Exec[exec run simple php script and segfault it]/returns: change from notrun to 0 failed: '
mkdir -p /tmp/749d71b922190d33e0314e614b1ecbd3 && \
cd /tmp/749d71b922190d33e0314e614b1ecbd3 && \
echo \#\!/bin/bash'
''
'\(php\ -r\ \'sleep\(10\)\;\'\ \&\)\ \&\&\ sleep\ 1\ \&\&\ kill\ -11\ \$\(pgrep\ php\)'
' > 749d71b922190d33e0314e614b1ecbd3 && \
chmod +x 749d71b922190d33e0314e614b1ecbd3 && \
./749d71b922190d33e0314e614b1ecbd3' returned 2 instead of one of [0]
Notice: /Stage[main]/Main/Node[default]/Helper::Script[run simple php script and segfault it]/Exec[cleanup run simple php script and segfault it]: Dependency Exec[exec run simple php script and segfault it] has failures: true
Warning: /Stage[main]/Main/Node[default]/Helper::Script[run simple php script and segfault it]/Exec[cleanup run simple php script and segfault it]: Skipping because of failed dependencies
Notice: Applied catalog in 1.05 seconds
Re-Running php5:extension:newrelic for Debian-8
Failure! 
```

**Comment by zazabe** [comment]
Still failing...

**Comment by ppp0** [comment]
retest this please

**Pull request merged**